### DISCUSSION TITLE
I have error on macOS

### DISCUSSION THREAD

**Discussion by ibragruslan** [CosmosOS/Cosmos#2880] [Q&A]
I install Cosmos on macOS Visual Studio, but I have problem.  I have error CS0246. When I start building the project, I get an error: NU1101. Please, help me

**Comment by zarlo** [comment]
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0246

https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1101
looks like it cant find a nuget package im guessing its the cosmos ones are they in (the nuget packages) /opt/cosmos
